### ISSUE TITLE
CLI orders api: per-order zip ends up being per-bundle

### ISSUE THREAD

**Issue by jreiberkyle** [planetlabs/planet-client-python#216]
When using the CLI to specify per-order zip, the order created ends up specifying per-bundle zip.

Example:
```
planet orders create \
  --id 20190415_170304_85_1068,20190415_170302_79_1068,20190426_170322_82_105c,20190426_170320_32_105c \
  --item-type psscene4band \
  --bundle analytic_sr \
  --zip order \
  --tools tools.json \
  --name tutorial_order
```

Gives:
```
{
  ...
  "delivery": {
    "archive_filename": "{{name}}_{{order_id}}.zip",
    "archive_type": "zip"
  },
...
}
```

This is missing the `"single_archive": true,` specification in `delivery` that would result in a per-order zip. Accordingly, when I download the order I end up getting one archive per id.

**Comment by sarasafavi** [comment]
this was actually fixed just last week, in [1.4.6](https://github.com/planetlabs/planet-client-python/releases/tag/1.4.6) -- @jreiberkyle can you confirm if you still see the bugged behavior in the latest release?

**Comment by jreiberkyle** [comment]
yep. checked and it's been fixed! I'll update the planet notebooks docker image
